doc_classes/MetaPlatformSDK_ChallengeEntry.xml

```diff
@@ -12,10 +12,11 @@
 		<member name="display_score" type="String" setter="" getter="get_display_score" default="&quot;&quot;">
 			A displayable score for this challenge entry. The score is formatted with thousands separators and the relevant units are appended based on the associated leaderboard's score type.
 		</member>
-		<member name="extra_data" type="String" setter="" getter="get_extra_data" default="&quot;&quot;">
+		<member name="extra_data" type="PackedByteArray" setter="" getter="get_extra_data" default="PackedByteArray()">
```

Comment:
I think this is fine, but won't actually be true until PR https://github.com/godot-sdk-integrations/godot-meta-toolkit/pull/7 is merged. Doesn't really matter since I think both will be merged at about the same time